client/src/components/TinymceInlineToolbar/TinymceInlineToolbar.js

```diff
@@ -161,21 +161,41 @@ class TinymceInlineToolbar {
  * @return {Boolean}
  */
 function shouldShowToolbar(selection, isEditorFocused, tagTypes) {
-  const tags = Array.isArray(tagTypes) ? tagTypes : [tagTypes || ''];
-  return Boolean(
-    selection &&
-    tags
-      .map(tag => String(tag))
-      .map(tag => tag.toLowerCase())
-      .includes((selection.tagName || '').toLowerCase()) &&
-    isEditorFocused);
+  let tags = Array.isArray(tagTypes) ? tagTypes : [tagTypes || ''];
```

Comment:
looks like you could move this `tags` definition to inside the below `if` statement

at the top of that statement you could have a `if (!tagTypes) { return true; }` at the top, so it doesn't need to go through all that map and filter chain